### PR TITLE
Adding sarif to pull requests

### PR DESCRIPTION
**Pull request by fjquileze**

### CODE REVIEW

**Comment by dagonco** [comment]
You've invoked whole Android team! Adding once @telefonica-android should be capable enough to add 2 reviewers randomly.

image

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -11,4 +11,11 @@ jobs:
     - name: "Build Android project"
       uses: docker://docker.tuenti.io/android/novum_android_api31:2
       with:
-        args: 'bash ./gradlew clean check assemble'
+        args: 'bash ./gradlew clean check assemble lint'
```
Comment (dagonco) [inline .github/workflows/tests.yml:14]: I can't tell right now if linter is already being executed with `check` task. But probably it is. Can you double check that?

**Review by dagonco — APPROVED** [review]
Great first PR! Let's go! 🚀

**Review by gmerinojimenez — APPROVED** [review]
Nice work!! congrats!

**Comment by gmerinojimenez** [comment]
> You've invoked whole Android team! Adding once @telefonica-android should be capable enough to add 2 reviewers randomly.

I think that was my bad 🤨

**Review by DevPabloGarcia — APPROVED** [review]
Nice first PR ! 🔥

**Pull request merged**